ingresses/production/ubuntu-com-internal.yaml

```diff
@@ -1,29 +1,25 @@
 ---
 
 kind: Ingress
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1beta1  # See https://bit.ly/2KdOtrZ
 metadata:
-  name: ubuntu-com
+  name: ubuntu-com-internal
   namespace: production
   annotations:
     kubernetes.io/ingress.class: "nginx"
     nginx.ingress.kubernetes.io/configuration-snippet: |
-      if ($host != 'ubuntu.com' ) {
+      if ($host == 'www.ubuntu.com' ) {
```

Comment:
Needs to be `if ($host = 'www.ubuntu.com' )`.

Reply:
Ooh. Awesome.

Reply:
Done.